CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+[xo-web/host] displaying host page even if controller aren't defined
```

Comment:
What's the visual issue?

And please add the PR link.

Reply:
When controller is undefined, the host component is "crashing" and the message `an error has occurred` appear on the interface.